sample/wallet/src/main/kotlin/com/walletconnect/sample/wallet/ui/routes/composable_routes/connections/ConnectionsRoute.kt

```diff
@@ -51,7 +51,9 @@ fun ConnectionsRoute(navController: NavController, connectionsViewModel: Connect
     val activity = context.findActivity()
 
     activity?.intent.takeIf { intent -> intent?.action == Intent.ACTION_VIEW && !intent.dataString.isNullOrBlank() }?.let { intent ->
-        web3WalletViewModel.pair(intent.dataString.toString())
+        if (intent.dataString?.startsWith("kotlin-web3wallet://wc") == true) {
```

Comment:
```suggestion
        if (intent.dataString?.startsWith("kotlin-web3wallet://wc?uri"") == true) {
```
It's due to the fact that web3modal on web uses `kotlin-web3wallet://wc?requestId` format for session requests